code/game/gamemodes/changeling/powers/augmented_eyesight.dm

```diff
@@ -12,54 +12,25 @@
 /obj/effect/proc_holder/changeling/augmented_eyesight/sting_action(mob/living/carbon/human/user)
 	if(!istype(user))
 		return
-	if(user.getorgan(/obj/item/organ/cyberimp/eyes/thermals/ling))
-		user << "<span class='notice'>Our eyes are protected from flashes.</span>"
-		var/obj/item/organ/cyberimp/eyes/O = new /obj/item/organ/cyberimp/eyes/shield/ling()
-		O.Insert(user)
-
+	var/obj/item/organ/eyes/E = user.getorganslot("eye_sight")
+	if(E)
+		if(E.flash_protect)
+			E.sight_flags |= SEE_MOBS
```

Comment:
so:
1. activate augmented eyesight power
2. remove own eyes (potentially use a friend)
3. turn the power off
4. insert the new eyes
5. permanent version of the power, respec to get around it.

are my fears unfounded?